player.js

```diff
@@ -0,0 +1,18 @@
+module.exports = class Player {
```

Comment:
Probably want to keep track of their user_id here too

Reply:
Since we're storing the user_id as a key for each player, this comment may no longer apply.